base/error.jl

```diff
@@ -93,6 +79,38 @@ function catch_backtrace()
     ret
 end
 
+"""
+    backtrace()
+
+Get a backtrace object for the current program point.
+"""
+function backtrace()
+    bt, bt2 = ccall(:jl_backtrace_from_here, Any, (Int32,), false)
+    if length(bt) > 2
+        # remove frames for jl_backtrace_from_here and backtrace()
+        if bt[2] == Ptr{Void}(-1%UInt)
+            # backtrace() is interpreted
+            deleteat!(bt, 1:3)
+            unshift!(bt2)
+        else
+            deleteat!(bt, 1:2)
```

Comment:
@JeffBezanson this is what's causing the win32 failure. On win32 the caller of this function is the topmost frame returned from backtrace_from_here.

Reply:
Ah, https://bugs.chromium.org/p/crashpad/issues/detail?id=53